packages/flutter/lib/src/rendering/box.dart

```diff
@@ -1587,9 +1587,6 @@ abstract class RenderBox extends RenderObject {
   @override
   BoxConstraints get constraints => super.constraints;
 
-  // We check the intrinsic sizes of each render box once by default.
-  bool _debugNeedsIntrinsicSizeCheck = true;
```

Comment:
In tests I figured we'd just be super aggressive and always test everything. It found a bug! (Sort of. The bug only existed because of the checks.) The 1400 tests still run in under 2 minutes, so the cost seems low.